src/components/AdmissionsDash.js

```diff
@@ -111,16 +112,31 @@ export class AdmissionsDash extends React.Component {
     this.fetchStudents();
   };
 
-  changeStudentStage = (option) => {
-    this.value = { value: option.value, label: allStages[option.value] };
+  changeStudentStage = (selectedOption) => {
+    this.setState({
+      selectedOption,
+    });
+    console.log(selectedOption, "selectedOption");
     const { filterValues } = this.state;
-    if (option.value === "default") {
+    if (selectedOption === null) {
       this.stage = null;
       this.dataType = "softwareCourse";
       this.fetchStudents(filterValues);
       this.value = "Student Details";
     } else {
-      this.stage = option.value;
+      // const arr = [];
+      let arr = selectedOption.map((option) => {
```

Comment:
Why can't this be `const`?
```suggestion
      const arr = selectedOption.map((option) => {
```